examples/data-objects/badge/src/BadgeModel.tsx

```diff
@@ -60,7 +60,7 @@ export class Badge extends DataObject implements IBadgeModel, IFluidHTMLView {
         const badgeHistory = SharedObjectSequence.create<IBadgeHistory>(this.runtime);
         badgeHistory.insert(0, [{
             value: current.get(),
-            timestamp: new Date(),
+            timestamp: new Date().toJSON(),
```

Comment:
@anthony-murphy - FYI - Legit bug caught by AsSerializable:

JSON.stringify(date) -> <ISO UTC string>
JSON.parse(<ISO UTC string>) -> <ISO UTC string>

... we almost get away with it.  See below...